app/src/main/kotlin/org/eurofurence/connavigator/pref/RemotePreferences.kt

```diff
@@ -58,18 +58,17 @@ object RemotePreferences : AnkoLogger {
     val timeSinceLastUpdate: DateTime get() = DateTime.now().minus(lastUpdatedMillis)
 
     // Booleans
-    val mapsEnabled: Boolean get() = remoteConfig.getBoolean("maps_enabled")
-    val rotationEnabled: Boolean get() = remoteConfig.getBoolean("rotation_enabled")
-    val nativeFursuitGames: Boolean get() = remoteConfig.getBoolean("native_fursuit_games")
-    val autoUpdateDisabled: Boolean get() = remoteConfig.getBoolean("auto_update_disabled")
+    val mapsEnabled: Boolean get() = remoteConfig.getBoolean("${BuildConfig.CONVENTION_IDENTIFIER}_maps_enabled")
```

Comment:
It would be more readable if we start using firebase remote conditions instead of doing this by hand. That way we keep backwards compatibility.